### ISSUE TITLE
Could not find a declaration file for module 'react-tinacms-editor'

### ISSUE THREAD

**Issue by jamespohalloran** [tinacms/tinacms#1573]
## Description

When importing from react-tinacms-editor, I get:

`Could not find a declaration file for module 'react-tinacms-editor'.`

### Steps to reproduce

`import { MarkdownFieldPlugin } from "react-tinacms-editor";`

### Expected result

Types should be imported correctly, and build succeeds without having to `//@ts-ignore`

### Actual result

Typescript build failed

#### Tinacms Package Versions

```text
├─┬ react-tinacms-editor@0.32.0
├─┬ react-tinacms-inline@0.32.0
├─┬ tinacms@0.32.0
│ ├── @tinacms/alerts@0.32.0
│ ├── @tinacms/core@0.32.0
│ ├─┬ @tinacms/fields@0.32.0
│ ├─┬ @tinacms/form-builder@0.32.0
│ ├─┬ @tinacms/forms@0.32.0
│ ├── @tinacms/icons@0.31.0
│ ├── @tinacms/react-alerts@0.32.0
│ ├─┬ @tinacms/react-core@0.32.0
│ ├─┬ @tinacms/react-forms@0.32.0
│ ├── @tinacms/react-modals@0.32.0
│ ├── @tinacms/react-screens@0.32.0
│ ├─┬ @tinacms/react-sidebar@0.32.0
│ ├─┬ @tinacms/react-toolbar@0.32.0
│ ├─┬ @tinacms/styles@0.29.0
```

NextJS site
Mac OSX.

It's a private repo, but ping me if you'd like access

**Comment by DirtyF** [comment]
Did you register the plugin and load the plugin dynamically?
https://tinacms.org/packages/react-tinacms-editor/#registering-the-plugins

**Comment by dwalkr** [comment]
It looks like this bug was introduced when we standardized the build scripts for all packages https://github.com/tinacms/tinacms/pull/1512/files

the typedef path should have been updated for react-tinacms-editor, but it was over looked. This should fix that https://github.com/tinacms/tinacms/pull/1575

**Comment by dwalkr** [comment]
@jamespohalloran try installing `react-tinacms-editor@next` and let me know if the problem persists

**Comment by jamespohalloran** [comment]
Yep that seems to have done the trick! Just noticed the issue is also happening for `next-tinacms-github`. Not sure about any other packages

**Comment by croutonn** [comment]
`next-tinacms-json` has also been overlooked.

**Comment by dwalkr** [comment]
@all-contributors please add @croutonn for bugs

**Comment by dwalkr** [comment]
I audited all the packages touched by that pull request and it looks like those two are the only ones that still had bad typedef paths. #1576 should fix this and I'll cut a release once that's merged

**Comment by dwalkr** [comment]
released in 0.32.1
